mlrun/model_monitoring/model_monitoring_batch.py

```diff
@@ -811,12 +812,20 @@ def run(self):
                     "hellinger_mean": drift_result["hellinger_mean"],
                 }
 
-                self.frames.write(
-                    backend="tsdb",
-                    table=self.tsdb_path,
-                    dfs=pd.DataFrame.from_dict([tsdb_drift_measures]),
-                    index_cols=["timestamp", "endpoint_id", "record_type"],
-                )
+                try:
+                    self.frames.write(
+                        backend="tsdb",
+                        table=self.tsdb_path,
+                        dfs=pd.DataFrame.from_dict([tsdb_drift_measures]),
+                        index_cols=["timestamp", "endpoint_id", "record_type"],
+                    )
+                except v3io_frames.errors.Error:
+                    logger.warn(
+                        "Events were not written to the TSDB: No TSDB schema file found at the target path, "
+                        "probably due to not enough model events",
+                        tsdb_path=self.tsdb_path,
+                        endpoint=endpoint_id,
+                    )
```

Comment:
How could you rely that the issue was that "no TSDB schema file found" was the issue based on the error you are catching, it seems like the error is generic error, to be more specific I would suggest to search for specific text inside of the error msg

Reply:
Right, I've updated the warning message to includes the general error (which in 99.9% cases is due to missing TSDB schema). 
![image](https://user-images.githubusercontent.com/101177563/210789381-175293fa-e85d-4e0f-998c-bd66505c6598.png)